amass/config.go

```diff
@@ -48,6 +48,9 @@ type AmassConfig struct {
 	// A blacklist of subdomain names that will not be investigated
 	Blacklist []string
 
+	// List of services that should not be run
+	DisabledServices []string
```

Comment:
@caffix are these config variables accessible from outside the amass package?

Reply:
Yes. Each enumeration has a config that is referenced throughout its execution.

I would prefer that the new configuration field be named DisabledDataSources

Reply:
Will do.